### PR TITLE
[FIX] Bug fixes for afni.model

### PR DESCRIPTION
**Pull request by emdupre**
Changes proposed in this pull request
- Patches cbucket output for 3dDeconvolve
- Correctly parses output of 3dSynthesize

### CODE REVIEW

**Comment by effigies** [comment]
Thanks for this. I'm canceling this CI job for the moment. I'll restart it when some high priority jobs finish.

File: nipype/interfaces/afni/model.py [hunk 1]
```diff
@@ -293,11 +293,14 @@ def _list_outputs(self):
...
 
+        if isdefined(self.inputs.cbucket):
+            outputs['cbucket'] = os.path.abspath(self.inputs.cbucket)
+
         outputs['reml_script'] = self._gen_fname(
             suffix='.REML_cmd', **_gen_fname_opts)
         # remove out_file from outputs if x1d_stop set to True
         if self.inputs.x1D_stop:
-            del outputs['out_file']
+            del outputs['out_file'], outputs['cbucket']
```
Comment (mvdoc) [inline nipype/interfaces/afni/model.py]: if I may, `test_extra_Deconvolve.py` could be extended to check for the removal of `outputs['cbucket']`

Comment (emdupre) [reply nipype/interfaces/afni/model.py]: Thanks for reviewing this, @mvdoc ! It sounds like a new release is imminent, but if @effigies doesn't mind me using a circle cycle I'll submit a patch to extend the test.

Comment (djarecka) [reply nipype/interfaces/afni/model.py]: @emdupre nothing is building right now and won't probably merge anything more. feel free to  use circle, we can always cancel.

Comment (effigies) [reply nipype/interfaces/afni/model.py]: Was going to say what @djarecka said. CI is not really the submitter's problem and we can cancel and restart as needed. I just try to give a heads up when doing that.

Comment (emdupre) [reply nipype/interfaces/afni/model.py]: Great, thanks for clarifying !

File: nipype/interfaces/afni/tests/test_extra_Deconvolve.py [hunk 2]
```diff
@@ -7,4 +7,5 @@ def test_x1dstop():
     deconv.inputs.out_file = 'file.nii'
     assert 'out_file' in deconv._list_outputs()
     deconv.inputs.x1D_stop = True
-    assert not 'out_file' in deconv._list_outputs()
\ No newline at end of file
+    assert not 'out_file' in deconv._list_outputs()
+    assert not 'cbucket' in deconv._list_outputs()
```
Comment (effigies) [inline nipype/interfaces/afni/tests/test_extra_Deconvolve.py]: The syntax `x not in y` is preferred to `not x in y`. Could you change both of these lines?

Comment (emdupre) [reply nipype/interfaces/afni/tests/test_extra_Deconvolve.py]: Patched and rebased into the previous commit!

**Comment by mvdoc** [comment]
LGTM 👍

**Pull request merged**